internal/node/node_launcher.sh

```diff
@@ -15,6 +15,30 @@
 
 set -e
 
+# --- begin runfiles.bash initialization ---
```

Comment:
Seems like not less code than before... 😥

Reply:
Sadly no. But at least its the recommended approach.